torch/_inductor/ir.py

```diff
@@ -4877,6 +4933,117 @@ def create(
         )
 
 
+class QLinearPointwisePT2E(ExternKernelAlloc):
```

Comment:
if possible, would be great to make this logic more concise, like what you did with quantized_lowerings. If not could we move this and other quantized lowerings to `quantized_ir.py` or something along those lines ? does not have to be in this pr

Reply:
Hi @eellison. Thanks for the suggestion. We think it difficult to make it more concise. We will move IR for quantized linear and conv to `quantized_ir.py` in a separate PR. How does that sound to you?